### ISSUE TITLE
AD/Domain integration

### ISSUE THREAD

**Issue by voxain** [Ylianst/MeshCentral#5443]
Hello thogether,
I couldn't find it anywhere in the manuals - So I'm asking here.
Is there a way to have MeshCentral integrated into AD in a way so that it automatically pulls things like PC descriptions from the AD?

We have the situation in our environment, that every PC is labelled with an incremental number - thus, finding the one PC that you need in a lost of PC0002, PC0034, PC0124 etc. is quite the problem. So it would be nice to be able to pull this information from the AD, where it is maintained.
Now I know that it is possible to connect MC to LDAP, but that'd only include the user logins as far as I'm aware.

**Screenshots**
![image](https://github.com/Ylianst/MeshCentral/assets/40141421/813cadfe-6101-44d0-9ba1-9f90eac29eaf)

**Server Software (please complete the following information):**
 - OS: Windows
 - Virtualization: Hyper-V
 - Network: LAN
 - Version: 1.1.13
 - Node: 20.4.0

**Client Device**
Not applicable

**Remote Device (please complete the following information):**
Not applicable

**Your config.json file**
Not applicable

**Comment by si458** [comment]
I don't believe this is possible at the moment as the descriptions are set per device and manually by the user

But what a great feature request! I'll look into it!

I'm thinking you would need 2 extra fields
Computer OU (so meshcentral knew where the computers would be located)
Description OU (so again meshcentral knew where the description comes from, ldap for example might be different than AD)

Also all ur computers would need to be under the same OU? Unless we can do a full search of the OU and sub OUs to find the computer name

**Comment by voxain** [comment]
That makes me glad to hear! We're currently searching for a replacement for our current remoting software, but that's one of the features which would be crucial to our workflow, so seeing it considered makes me glad!
Currently we have all our computers in a single OU with a few scattered around some sub OUs. Defining that would make life a whole lot easier, because there's also a lot of dead entries that haven't been gotten rid off of.

Also, while I'm here I wanna suggest that the search bar in the device overview should also search through the comments, as I don't think it's doing that right now. However, this is not crucial as it's easily replacable by a simple CTRL+F.

Thank you for taking your time!

**Comment by voxain** [comment]
Been a while! Any update? DameWare is starting to get a little.. frustrating.. :)

**Comment by voxain** [comment]
man

**Comment by voxain** [comment]
bump

**Comment by voxain** [comment]
bump
